src/foam/u2/layout/MDUpdateBrowserView.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'MDUpdateBrowserView',
+  extends: 'foam.u2.View',
+
+  documentation: 'MD-styled update browser',
+
+  topics: [
+    'finished',
+    'throwError'
+  ],
+
+  axioms: [
+    foam.pattern.Faceted.create()
+  ],
+
+  requires: [
+    'foam.u2.ControllerMode',
+    'foam.u2.ToolbarAction',
+    'foam.u2.layout.MDToolbarView'
+  ],
+
+  imports: [
+    'stack'
+  ],
+
+  exports: [
+    'controllerMode',
+    'as objectSummaryView'
+  ],
+
+  properties: [
+    {
+      class: 'FObjectProperty',
+      name: 'data'
+    },
+    {
+      name: 'controllerMode',
+      factory: function() {
+        return this.ControllerMode.CREATE;
+      }
+    },
+    {
+      class: 'foam.u2.ViewSpecWithJava',
+      name: 'viewView',
```

Comment:
I know you just copied this name 'viewView', but it doesn't seem like a good name.